### DISCUSSION TITLE
Add usercard for the author of the raid

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5357] [Ideas]
Chat message after a raid
18327 raiders from LIRIK have joined!

When a raider is not very popular, it is interesting who came in, but his nickname is not clickable. Currently, to go to the author of the raid, you either need to wait for his message, or enter his nickname manually in the browser.

I suggest making the author of the raid clickable in the same way as you can click on any chatters and see their usercard:
18327 raiders from **LIRIK** have joined!

**Comment by 8thony** [comment]
Similar to #2673
